InfoMenu.pm

```diff
@@ -325,7 +325,7 @@ sub menuBrowse {
 		} elsif ( $type eq 'playlist' ) {
 
 			Plugins::TIDAL::Plugin::getAPIHandler($client)->playlist(sub {
-				my $feed = Plugins::TIDAL::Plugin::_renderItem( $client, $_[0] ) if $_[0];
```

Comment:
Why is this required? Wouldn't `_renderItem()` recognise the data as a playlist? In that case maybe `_renderItem()` should be fixed? Or you could call `_renderTrack()` directly instead of `_renderItem()`?